imports/plugins/core/ui-tagnav/client/components/tagTree.js

```diff
@@ -0,0 +1,163 @@
+import React, { Component, PropTypes } from "react";
```

Comment:
@kieha I need little help with this component.... specifically making it share a number of functions from the container file above. I've done the usual passing them in as props, but much of them are not doing what is expected. 

Running this branch gives a clearer picture of what I'm saying.... so we can sync on this. My guess is that I'm making a React beginner mistake there.